content/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -48,7 +48,7 @@ The **Variables** page appears, showing all workspace-specific variables and var
 
 ### Add a Variable
 
-To add a variable:
+Terraform automatically detects undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values in your configuration when you [create a new workspace](/cloud-docs/workspaces/creating) using the VCS workflow. To add additional Terraform or environment variables:
```

Comment:
I thought about adding something like this but wasn't sure if it's unnecessary:

```suggestion
Terraform automatically detects undefined [Terraform variables](/cloud-docs/workspaces/variables#terraform-variables) without default values in your configuration when you [create a new workspace](/cloud-docs/workspaces/creating) using the VCS workflow. To add additional Terraform or environment variables (eg., set a value for a Terraform variable defined in your configuration):
```

Reply:
I see what you are thinking....hmmmm i don't know that this is necessary here though. I think we should leave it out and see.